tests/test_sliding_window.py

```diff
@@ -207,14 +217,15 @@ def test_single_prefill_sliding_window(
     torch.testing.assert_close(o.cpu(), o_ref.cpu(), rtol=1e-3, atol=1e-3)
 
 
-@pytest.mark.parametrize("batch_size", [12, 17])
-@pytest.mark.parametrize("kv_len", [54, 397])
-@pytest.mark.parametrize("qo_len", [37, 47])
-@pytest.mark.parametrize("window_left", [13, 33])
-@pytest.mark.parametrize("num_kv_heads", [1, 4])
-@pytest.mark.parametrize("num_qo_heads", [4, 8])
+@pytest.mark.parametrize("batch_size", [12, 17, 71])
+@pytest.mark.parametrize("kv_len", [54, 397, 1177, 2999])
+@pytest.mark.parametrize("qo_len", [1, 37, 47])
+@pytest.mark.parametrize("window_left", [13, 33, 111])
+@pytest.mark.parametrize("num_kv_heads", [32])
+@pytest.mark.parametrize("num_qo_heads", [32])
```

Comment:
Please don't remove the GQA case, changing sliding windows' behavior might influence GQA because of the use of GQA packing.